src/view.rs

```diff
@@ -242,6 +242,7 @@ impl<'a> View<'a> {
         _ => {
           // Nothing in the buffer. Wait for a bit...
           std::thread::sleep(std::time::Duration::from_millis(100));
+          continue; // don't render again if nothing new to show
```

Comment:
Nice! Also, I'm going to tweak a bit another case that flickers. When you have many keystrokes in the stdin (pushing long keys), we are saying to `termion` to print many things. I'm going to throttle this.

Reply:
https://github.com/fcsonline/tmux-thumbs/commit/10eff07d160bd53c0949fcb66e623687ba01b668